src/applications/dispute-debt/config/form.js

```diff
@@ -0,0 +1,116 @@
+import footerContent from 'platform/forms/components/FormFooter';
+import { VA_FORM_IDS } from 'platform/forms/constants';
+
+import profileContactInfo from 'platform/forms-system/src/js/definitions/profileContactInfo';
+import {
+  veteranInformation,
+  debtSelection,
+  disputeReason,
+  supportStatement,
+} from '../pages';
+
+import ConfirmationPage from '../containers/ConfirmationPage';
+import IntroductionPage from '../containers/IntroductionPage';
+
+import prefillTransformer from './prefill-transformer';
+import { TITLE, SUBTITLE } from '../constants';
+import manifest from '../manifest.json';
+
+/** @type {FormConfig} */
+const formConfig = {
+  rootUrl: manifest.rootUrl,
+  urlPrefix: '/',
+  submitUrl: '/debts_api/v0/digital_disputes',
+  submit: () =>
+    Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+  trackingPrefix: 'dispute-debt',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  dev: {
+    showNavLinks: true,
+    collapsibleNavLinks: true,
+  },
+  formId: VA_FORM_IDS.FORM_DISPUTE_DEBT,
+  saveInProgress: {
+    // messages: {
```

Comment:
you want these comments here? NBD, feel free to resolve this.

Reply:
Yup! Left on purpose, we have follow on tickets to add these props. Thanks for spotting it!